### DISCUSSION TITLE
module 'tweepy' has no attribute 'Client'

### DISCUSSION THREAD

**Discussion by luckyonetwo** [tweepy/tweepy#1864] [Questions] [answered]
how to deal with the problem of module 'tweepy' has no attribute 'Client'

**Comment by Harmon758** [comment, marked as answer]
You're probably either using an older version of Tweepy (`tweepy.Client` was added with support for Twitter API v2 in Tweepy v4.0.0) or have another file or folder in your path named "tweepy".

Regardless, this is not a reproducible issue with Tweepy and there's nothing actionable here.
